platform/darwin/src/MGLStyle.h

```diff
@@ -158,7 +164,45 @@ static const NSInteger MGLStyleDefaultVersion = 9;
  */
 + (NSURL *)satelliteStreetsStyleURLWithVersion:(NSInteger)version;
 
-- (instancetype)init NS_UNAVAILABLE;
+#pragma mark Runtime Styling
+
+/**
+ Returns a layer that conforms to MGLStyleLayer if any layer with the given identifier was found.
```

Comment:
We’ve recently settled on wrapping these lines at 80 characters. Xcode has a convenient “Page guide at column” setting at in Text Settings.